src/components/Carousel/index.tsx

```diff
@@ -0,0 +1,45 @@
+'use client'
+import { CAROUSEL_ITEMS } from '@/constants'
+import Image from 'next/image'
+import { Carousel as BsCarousel } from 'react-bootstrap'
+
+export function Carousel() {
+  return (
+    <div>
```

Comment:
Remove this wrapper div please. Sorry I missed this one earlier

Reply:
done